From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ali Mohammad Pur <ali.mpfard@gmail.com>
Date: Wed, 12 Jan 2022 22:18:23 +0330
Subject: [PATCH] cmcurl: Use struct stat and include sys/stat.h

For unknown reasons, curl_setup_once.h does not include sys/stat.h. This patch includes sys/stat.h.

- [ ] Local?
- [ ] Should be merged to upstream?
- [X] Resolves issue(s) with our side of things
- [X] Hack
---
 Utilities/cmcurl/lib/curl_setup.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Utilities/cmcurl/lib/curl_setup.h b/Utilities/cmcurl/lib/curl_setup.h
index 7421b670..b06c19d8 100644
--- a/Utilities/cmcurl/lib/curl_setup.h
+++ b/Utilities/cmcurl/lib/curl_setup.h
@@ -403,6 +403,7 @@
 #endif
 
 #ifndef struct_stat
+#  include <sys/stat.h>
 #  define struct_stat struct stat
 #endif
 
